Task 6/Task 6.3/style.css

```diff
@@ -0,0 +1,35 @@
+body {
+    font-size: 18px;
+    background-color: #333333;
+}
+
+.mainBlock{
+    border: 5px solid #c2e1f5;
+    text-align: center;
+    margin: auto;
+    margin-top: 70px;
+    background: #88b7d5;
+    min-width: 300px;
+}
+
+.mainBlock::before {
+    content: "";
+    border-top: 50px solid transparent;
+    border-left: 50px solid transparent;
+    border-right: 50px solid transparent;
+    border-bottom: 50px solid #c2e1f5;
+    position: absolute;
+    top: -30px;
+    margin-left: -50px;
+}
```

Comment:
since you don't have `position:relative` in your parent element (element is a parent to its pseudo elements), pseudo elements are positioning relative to a page. If I move the element, triangles won't move after the element.